src/pfe/portal/modules/User.js

```diff
@@ -213,49 +213,28 @@ module.exports = class User {
    * Function to get watchList for all open projects
    */
   async getWatchList() {
```

Comment:
Might be worth splitting this out to the watchlist API route (in a separate function so we can unittest it still).